hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -492,7 +488,7 @@ public static class Builder extends WithParentObjectId.Builder<OmKeyInfo> {
     private FileChecksum fileChecksum;
 
     private boolean isFile;
-    private final Map<String, String> tags = new HashMap<>();
+    private MapBuilder<String, String> tags = MapBuilder.empty();
```

Comment:
Let's not eagerly create an empty one.

```suggestion
    private final MapBuilder<String, String> tags;
```